README.md

```diff
@@ -2,111 +2,78 @@
-| `vote_application_threshold` | `3` | Number of same votes required for immediate application of an edit. Set to zero to disable automatic application. |
-| `voting_period` | `345600` | Time, in seconds, before a voting period is closed. |
-| `min_destructive_voting_period` | `172800` | Minimum time, in seconds, that needs to pass before a destructive edit can be immediately applied with sufficient positive votes. |
-| `vote_cron_interval` | `5m` | Time between runs to close edits whose voting periods have ended. |
-| `email_host` | (none) | Address of the SMTP server. Required to send emails for activation and recovery purposes. |
-| `email_port` | `25` | Port of the SMTP server. |
-| `email_user` | (none) | Username for the SMTP server. Optional. |
-| `email_password` | (none) | Password for the SMTP server. Optional. |
-| `email_from` | (none) | Email address from which to send emails. |
-| `host_url` | (none) | Base URL for the server. Used when sending emails. Should be in the form of `https://hostname.com`. |
-| `image_location` | (none) | Path to store images, for local image storage. An error will be displayed if this is not set when creating non-URL images. |
-| `image_backend` | (`file`) | Storage solution for images. Can be set to either `file` or `s3`. |
-| `userLogFile` | (none) | Path to the user log file, which logs user operations. If not set, then these will be output to stderr. |
-| `s3.endpoint` | (none) | Hostname to s3 endpoint used for image storage. |
-| `s3.base_url` | (none) | Base URL to access images in S3. Should be in the form of `https://hostname.com`. |
-| `s3.bucket` | (none) | Name of S3 bucket used to store images. |
-| `s3.access_key` | (none) | Access key used for authentication. |
-| `s3.secret ` | (none) | Secret Access key used for authentication. |
-| `s3.max_dimension` | (none) | If set, a resized copy will be created for any image whose dimensions exceed this number. This copy will be served in place of the original.
-| `phash_distance` | 0 | Determines what binary distance is considered a match when querying with a phash fingeprint. Using more than 8 is not recommended and may lead to large amounts of false positives. **Note**: The [pg-spgist_hamming extension](#phash-distance-matching) must be installed to use distance matching, otherwise you will get errors. |
-| `favicon_path` | (none) | Location where favicons for linked sites should be stored. Leave empty to disable. |
-| `draft_time_limit` | (24h) | Time, in seconds, before a draft is deleted. |
-| `profiler_port` | 0 | Port on which to serve pprof output. Omit to disable entirely. |
-| `postgres.max_open_conns` | (0) | Maximum number of concurrent open connections to the database. |
-| `postgres.max_idle_conns` | (0) | Maximum number of concurrent idle database connections. |
-| `postgres.conn_max_lifetime` | (0) | Maximum lifetime in minutes before a connection is released. |
```

Comment:
A whole section is being removed that explains what different configuration keys in `stash-box-config.yml` means.